<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Slave Password Generator</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="0">
     <widget class="QLabel" name="slaveKeyLabel">
      <property name="text">
       <string>Clé esclave</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="masterPasswordLabel">
      <property name="text">
       <string>Mot de passe maître</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="slavePasswordLabel">
      <property name="text">
       <string>Mot de passe esclave</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="masterPasswordField"/>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="slaveKeyField"/>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="slavePasswordField">
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>masterPasswordField</sender>
   <signal>textChanged(QString)</signal>
   <receiver>MainWindow</receiver>
   <slot>compute()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>171</x>
     <y>84</y>
    </hint>
    <hint type="destinationlabel">
     <x>142</x>
     <y>53</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>slaveKeyField</sender>
   <signal>textChanged(QString)</signal>
   <receiver>MainWindow</receiver>
   <slot>compute()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>222</x>
     <y>156</y>
    </hint>
    <hint type="destinationlabel">
     <x>198</x>
     <y>52</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>compute()</slot>
 </slots>
</ui>
